romancal/tests/test_dms_requirements.py

```diff
@@ -20,7 +20,7 @@ def test_requirements():
     }
 
     existing_tests = set()
-    test_regex = re.compile(r"def (test_[^\(]+)\(.*\):")
+    test_regex = re.compile(r"def (test_[^\(]+)\(")
```

Comment:
@zacharyburnett I modified the regex here to catch multi-line function signatures which won't end in `):`. Does this look good to you?

Reply:
Good catch! Yeah that looks good, it only needs the function name